readme.txt

```diff
@@ -74,6 +74,8 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 == Changelog ==
 
 == Unreleased ==
+
+- Fix: `@woocommerce/navigation.getIdsFromQuery` support for `0` as a valid id. #7028
```

Comment:
We don't need a changelog item for WC Admin as it doesn't directly impact it- i.e. it's not fixing a bug in WC Admin.